src/Hint/Fixities.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE ViewPatterns, ScopedTypeVariables #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeFamilies #-}
+{-
+
+Raise a warning if you have redundant brackets in nested infix expressions.
+
+<TEST>
+yes = 1 + (2 * 3) -- @Ignore 1 + 2 * 3
+no = (1 + 2) * 3
+no = 1 + 2 * 3
```

Comment:
Could we have another test of brackets in the other direction, e.g. `1 * 2 + 3`. A lot of the fixity stuff is left vs right, so an easy potential mistake.

Reply:
Done